hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/io/BaseTestHandle.java

```diff
@@ -55,13 +55,15 @@ public void tearDown() throws Exception {
   }
 
   Pair<WriteStatus, List<HoodieRecord>> createParquetFile(HoodieWriteConfig config, HoodieTable table, String partitionPath,
-                                                          String fileId, String instantTime, HoodieTestDataGenerator dataGenerator) {
+                                                          String fileId, String instantTime, HoodieTestDataGenerator dataGenerator,
+                                                          boolean preserveMetadata) {
     List<HoodieRecord> records = dataGenerator.generateInserts(instantTime, 100);
     Map<String, HoodieRecord> recordMap = new HashMap<>();
     for (int i = 0; i < records.size(); i++) {
       recordMap.put(String.valueOf(i), records.get(i));
     }
     HoodieCreateHandle handle = new HoodieCreateHandle(config, instantTime, table, partitionPath, fileId, recordMap, new LocalTaskContextSupplier());
+    handle.setPreserveMetadata(preserveMetadata);
```

Comment:
why can't we call the other overloaded constructor in L65 which takes in the `preserveMetadata` arg. 
why do we need this setter just in tests?

Reply:
Removed this change. I had added it for fixing one of the tests. Not required anymore.